### ISSUE TITLE
Upgrade Axios / Audit = High Severity (kendo-react-upload)

### ISSUE THREAD

**Issue by bdbvb** [telerik/kendo-react#2460]
## I'm submitting a...
* Security issue

## Current behavior
npm audit yields high severity issue related to axios:

```javascript
    "axios": {
      "name": "axios",
      "severity": "high",
      "isDirect": false,
      "via": [
        {
          "source": 1098582,
          "name": "axios",
          "dependency": "axios",
          "title": "Server-Side Request Forgery in axios",
          "url": "https://github.com/advisories/GHSA-8hc4-vh64-cxmj",
          "severity": "high",
          "cwe": [
            "CWE-918"
          ],
          "cvss": {
            "score": 0,
            "vectorString": null
          },
          "range": ">=1.3.2 <=1.7.3"
        }
      ],
      "effects": [
        "@progress/kendo-react-upload"
      ],
      "range": ">=1.3.2",
      "nodes": [
        "node_modules/axios"
      ],
      "fixAvailable": false
    },
``` 

## Expected behavior
no security issue

## Minimal reproduction of the problem with instructions
install @progress/kendo-react-upload, then run npm audit

## What is the motivation or use case for changing the behavior?
Security

## Environment

Package versions:

```
"@progress/kendo-react-upload": "8.2.0"
```

Browser:
N/A

System:
- TypeScript version: N/A
- Node version: 20.15.1, npm 10.8.2
- Platform:  Windows

**Comment by PaskalevStoyan** [comment]
Hi @bdbvb,

Thanks for bringing this issue to our attention. We are aware of the bug regarding axios and we are in the middle of fixing it. I will write back when we include it in the next version of the KendoReact Upload Package.

Best Regards,

Stoyan

**Comment by PaskalevStoyan** [comment]
Hello @bdbvb,

We bumped the axios version to 1.7.4 which resolves the vulnerability. Its included in `8.3.0-develop.5` version of the KendoReact Upload Package.
